src/lib/containers/download_list_v2/AvailableForDownloadTable.tsx

```diff
@@ -24,8 +29,13 @@ export default function AvailableForDownloadTable() {
   return (
     <QueryClientProvider client={queryClient}>
       <SynapseErrorBoundary>
-        <DownloadListTable />
+        {
+          props.filesStatistics && (
+            <DownloadListTable key={props.filesStatistics.totalNumberOfFiles} />
```

Comment:
This works. Do you mind putting a comment here to explain the key, even if it's just a link to the Jira issue?